Description: In case environment variable MNE_FORCE_SERIAL is set, the log
output has 3 lines causing the test to fail.
 .
 python-mne (0.10+dfsg-1.1) UNRELEASED; urgency=medium
 .
   * New upstream version
   * Removed depedency on mayavi2
Author: Jaakko Leppakangas <jaeilepp@student.jyu.fi>

--- python-mne-0.10+dfsg.orig/mne/tests/test_filter.py
+++ python-mne-0.10+dfsg/mne/tests/test_filter.py
@@ -132,9 +132,9 @@ def test_notch_filters():
             set_log_file()
             with open(log_file) as fid:
                 out = fid.readlines()
-            if len(out) != 2:
+            if len(out) != 2 and len(out) != 3:  # force_serial: len(out) == 3
                 raise ValueError('Detected frequencies not logged properly')
-            out = np.fromstring(out[1], sep=', ')
+            out = np.fromstring(out[-1], sep=', ')
             assert_array_almost_equal(out, freqs)
         new_power = np.sqrt(sum_squared(b) / b.size)
         assert_almost_equal(new_power, orig_power, tol)
